lara-typescript/src/lib/plugin-context.ts

```diff
@@ -14,12 +15,22 @@ export interface IPluginContext {
   pluginId: number;
   /** The authored configuration for this instance (if available). */
   authoredState: string | null;
+  /** Reserved HTMLElement for the plugin output. */
+  container: HTMLElement;
+  /** Wrapped embeddable runtime context if plugin is wrapping some embeddable. */
+  wrappedEmbeddable: IEmbeddableContextOptions | null;
```

Comment:
So this is added to the IPluginAuthoringContextOptions interface, but IEmbeddableContextOptions (previously IEmbeddableContext) has some fields that are Runtime specific: interactiveStateUrl and interactiveAvailable.  As far as I can tell the wrappedEmbeddable is not passed through to the IPluginAuthoringContext so it is probably best to just remove this from IPluginCommonOptions for now. 

We'll need to add something like it back in to support a real preview, but it will need a different type signature.

Reply:
We could remove it, but I think we'd add it right back in when supporting preview.

I think the `IEmbeddableContextOptions` type is ok. `interactiveStateUrl` is allowed to be `null`, and if `interactiveAvailable` refers to `clickToPlay` (I think it does), then we probably want to know that when rendering the preview, so that the teacher-edition model looks the same as it would look to the student.

But I don't feel strongly about this, so it could also be removed.

Reply:
I'm pretty sure the IEmbeddable* object that is needed for the preview will be different. We talked about having a route in LARA to render the html of the embeddable, so the plugin could use that. So then the wrappedEmbeddable object would contain a URL to this embeddable route. That URL wouldn't be something we'd want to expose to plugin runtime. Supporting click to play will be complicated with this preview approach since it requires not only html but also javascript which seems tricky to get right, so I'd guess we will just avoid it on first pass.

So I still think we should remove it for now, so we we don't feel we need to use this same interface when working on the preview.